queries.sql

```diff
@@ -149,4 +106,4 @@ LEFT JOIN specializations s ON s.vets_id=vets.id
 JOIN animals ON animals.species_id=s.species_id 
 JOIN species ON species.id=s.species_id 
 WHERE vets.name='Maisy Smith' 
-GROUP BY species.name ORDER BY count_species DESC LIMIT 1;
\ No newline at end of file
+GROUP BY species.name ORDER BY count_species DESC LIMIT 1;
```

Comment:
- [ ] Here for
-  * How many times visits where with a vet that did not specialize in that animal's species?

```sql
  SELECT count(*)
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE animals.species_id NOT IN (SELECT species_id FROM specializations WHERE vet_id = vets.id)
```

- What specialty should Maisy Smith consider getting? Look for the species she gets the most.

```sql
  SELECT species.name, count(*)
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN species ON animals.species_id = species.id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vet.name = 'Maisy Smith'
  GROUP BY species.name
```

Reply:
Thank you, this is the correct solution. I was don't understanding the question.

Reply:
I am adding solutions in the PR description

Reply:
Fixed.